test-specifications/cryptoService/accountCreateTransaction.md

```diff
@@ -0,0 +1,156 @@
+# AccountCreateTransaction - Test specification
+
+## Description:
+This test specification for AccountCreateTransaction is to be one of many for testing the functionality of the Hedera SDKs. The SDK under test will use the language specific JSON-RPC server return responses back to the test driver.
+
+## Design:
```

Comment:
> The inputs for each test are a range of valid, minimum, maximum, negative and invalid values for the method.

This is not always the case, so maybe update this sentence or make it always be the case.

Reply:
I added more tests to make this sentence valid

Reply:
Replied in another conversation on it.

Reply:
Added test cases for default cases

Reply:
Looks good, I added couple more comments to tests.